client-v2/src/shared/components/input/InputImage.tsx

```diff
@@ -19,7 +22,10 @@ const ImageContainer = styled('div')<{
   width: 100%;
   max-width: ${props => (props.size === 'small' ? '100px' : '180px')};
   height: ${props => (props.size === 'small' ? '60px' : '115px')};
-  background-color: ${props => (props.isHovering ? '#bbb' : '#ccc')};
+  background-color: ${props =>
+    props.isHovering
+      ? props.theme.base.colos.placeholderLight
```

Comment:
There looks like a typo here, which suggests that some types aren't getting through maybe??

Reply:
This has been fixed with the strongly typed theme swapped in throughout.

Reply:
this should be resolved by the changes @vlbee has made to imports (see conversation around BasePlaceholder.tsx), which will provide the right types 😄

Reply:
got there just before me! 👍